### ISSUE TITLE
Rename ConnectionSettings `close_after_match`

### ISSUE THREAD

**Issue by NicEastvillage** [RLBot/flatbuffers-schema#32]
The `ConnectionSettings::close_after_match` has confused a few people using the lower level interfaces since the connection is not actually closed when the match ends, but rather, when a match is stopped with a `StopMatch` message or a new match begins with a `StartMatch` message. I propose renaming it to `close_between_matches`.

**Comment by swz-git** [comment]
I think its pretty hard to define the behavior of this field with a short name. `close_between_matches` still implies that the bot will get closed after a match. The best we could do IMO is replace/append [this](https://github.com/RLBot/flatbuffers-schema/blob/a01a99dedbe9f7b58e3f91191dbd20e90c476d84/rlbot.fbs#L18) comment with something like `/// RLBot doesn't actually know when a match ends. Therefor, your bot/script will only be closed once a StopMatch or StartMatch packet is sent`.

**Comment by NicEastvillage** [comment]
Yeah, that comment would be a great addition to remove all confusion. But I still think `close_between_matches` is slightly better since it does not have `immediacy` to it.
